Control/Monad/Accum.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE CPP #-}
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Control.Monad.Accum
+-- Copyright   :  (c) Manuel Bärenz,
+-- License     :  BSD-style (see the file LICENSE)
+--
+-- Maintainer  :  libraries@haskell.org
+-- Stability   :  experimental
+-- Portability :  non-portable (multi-param classes, functional dependencies)
+--
+-- The MonadAccum class.
+--
+--      Inspired by the paper
```

Comment:
I think you should remove this section  given this transformer is not inspired by that paper. Some similar modules don't have this disclaimer in them, such as `Control.Monad.Identity`.